docs/user-guide/src/version_support.md

```diff
@@ -17,20 +17,22 @@ Cluster API Provider Metal3
 
 | Minor release | API version | Status    |
 | ------------- | ----------- | --------- |
+| v1.7          | v1beta1     | Supported |
 | v1.6          | v1beta1     | Supported |
-| v1.5          | v1beta1     | Supported |
-| v1.4          | v1beta1     | Tested    |
+| v1.5          | v1beta1     | Tested    |
+| v1.4          | v1beta1     | EOL       |
 | v1.3          | v1beta1     | EOL       |
-| v1.2          | v1beta1     | EOL       |
-| v1.1          | v1beta1     | EOL       |
+| v1.2          | v1alpha1    | EOL       |
+| v1.1          | v1alpha1    | EOL       |
 
 IP Address Manager
 
 | Minor release | API version | Status    |
 | ------------- | ----------- | --------- |
+| v1.7          | v1beta1     | Supported |
 | v1.6          | v1beta1     | Supported |
-| v1.5          | v1beta1     | Supported |
-| v1.4          | v1beta1     | Tested    |
+| v1.5          | v1beta1     | Tested    |
+| v1.4          | v1beta1     | EOL       |
```

Comment:
It should be same v1beta1 for v1.2 and 1.1

Reply:
Oops. Sorry.